### PR TITLE
Elastic Load Balancer support

### PR DESCRIPTION
**Pull request by egor045**
Add ELB support:
- New elb component
- New elb API factory
- Add elb entries to routes.js
- Add elb detail support to factories/utils.js

### CODE REVIEW

File: source/javascript/components/elb/list.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<section class="md-whiteframe-2dp md-padding">
+    <filters filters="vm.filters" on-update="vm.updateFilters" on-reset="vm.resetFilters"></filters>
+
+    <md-table-pagination
+            style="border: 0;"
+            md-on-paginate="vm.updatePath"
+            md-page="vm.params.page"
+            md-page-select
+            md-limit="vm.params.count"
+            md-limit-options="[10, 20, 50]"
```
Comment (TheM0ng00se) [inline source/javascript/components/elb/list.html]: per comments on the collector if we can go with 25,50,100 for the counts would be great.

File: source/javascript/components/elb/list.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            </tbody>
+        </table>
+    </md-table-container>
+
+    <md-table-pagination
+            md-on-paginate="vm.updatePath"
+            md-page="vm.params.page"
+            md-page-select
+            md-limit="vm.params.count"
+            md-limit-options="[10, 20, 50]"
```
Comment (TheM0ng00se) [inline source/javascript/components/elb/list.html]: same as above

File: source/javascript/routes.js [hunk 3]
```diff
@@ -871,6 +871,47 @@ function config($stateProvider, $urlServiceProvider) {
...
+        .state('elb', {
+            abstract: true,
+            parent: 'main',
+            template: '<ui-view/>'
+        })
+        .state('elb.list', {
+            url: '/elb/list?{page:int}&{count:int}&{accounts:string}&{regions:string}&{numInstances:int}',
+            params: {
+                page: 1,
+                count: 20,
```
Comment (TheM0ng00se) [inline source/javascript/routes.js]: standard is 100 for this.

**Review by TheM0ng00se — CHANGES_REQUESTED** [review]

**Comment by egor045** [comment]
Counts updated to [25, 50, 100]

**Review by TheM0ng00se — APPROVED** [review]

**Pull request merged**